### ISSUE TITLE
Time Control Module Doesnt save date  correctly 

### ISSUE THREAD

**Issue by harshdoshihd** [YetiForceCompany/YetiForceCRM#13978]
Hi Team,

I am trying to save user activity in time control as:

Scenario 1:
Start date: 28th Dec 2020 Start time: 10:00
End Date: 28th Dec 2020 End Time: 1800

Result: Start date: 29th Dec 2020 Start time 10:00
End Date 28th Dec 2020 End Time 18:00

Scenario 2:
Start date: 28th Dec 2020 Start time: 13:00
End Date: 28th Dec 2020 End Time: 1800

Result: Start date: 28th Dec 2020 Start time 10:00
End Date 28th Dec 2020 End Time 18:00

![timecontrol](https://user-images.githubusercontent.com/61234565/103226203-966f7e80-4951-11eb-8afd-bc1d431d338a.png)

This is version 6.1

My previous set-up had version 5.3 there its working fine

**Comment by mariuszkrzaczkowski** [comment]
![github](https://user-images.githubusercontent.com/10198654/103242194-ab4e1100-4955-11eb-81ad-31bf9ab7c1df.jpg)

**Comment by mariuszkrzaczkowski** [comment]
1.attach the missing information
2. what is the user's time zone

**Comment by harshdoshihd** [comment]
Hi Marius,

The real problem was that during installation I set the time zone as default - New York, the users time zone was set to India (+5:30hrs). Hence the issue popped up.

But, in our case all the users are from India hence, I just changed the default time zone to India - IST. Hence the issue was solved.

But, for any organization who have employees in different time zone, there could be an issue where the server will only be showing default time zone set-during installation. and it will clash between two different timezone (user timezone and default installation time zone).

Thank you and Happy New Year Ahead!
